instrumentation/netty/netty-4-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/common/client/NettyClientInstrumenterFactory.java

```diff
@@ -46,12 +46,12 @@ public NettyClientInstrumenterFactory(
         .newClientInstrumenter(new HttpRequestHeadersSetter());
   }
 
-  public NettyConnectInstrumenter createConnectInstrumenter() {
+  public NettyConnectionInstrumenter createConnectionInstrumenter() {
     NettyConnectNetAttributesExtractor netAttributesExtractor =
         new NettyConnectNetAttributesExtractor();
-    Instrumenter<NettyConnectRequest, Channel> instrumenter =
-        Instrumenter.<NettyConnectRequest, Channel>builder(
-                GlobalOpenTelemetry.get(), instrumentationName, rq -> "CONNECT")
+    Instrumenter<NettyConnectionRequest, Channel> instrumenter =
+        Instrumenter.<NettyConnectionRequest, Channel>builder(
+                GlobalOpenTelemetry.get(), instrumentationName, NettyConnectionRequest::spanName)
```

Comment:
I sort of see these as two different instrumenters, one for resolve and one for connect, but ok with this too

Reply:
That was my first idea too, but then I noticed that they're basically the same; so I ended up with just reusing it and changing just the span name.